src/ray/raylet/placement_group_resource_manager.h

```diff
@@ -56,6 +56,13 @@ class PlacementGroupResourceManager {
   /// \param bundle_spec: Specification of bundle whose resources will be prepared.
   virtual bool PrepareBundle(const BundleSpecification &bundle_spec) = 0;
 
+  /// Prepare resource batched, it's an encapsulation of function `PrepareBundle`.
+  ///
+  /// \param bundle_specs: A set of bundles that waiting to be prepared.
```

Comment:
```suggestion
  /// \param bundle_specs A set of bundles that waiting to be prepared.
```
We don't have colon in our docstring (see other functions in other modules)

Reply:
updated.